tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -488,6 +493,46 @@ func (a *AwsProvider) createInstance(imageID, instanceType, sshKey string,
 	return runResult.Instances[0], nil
 }
 
+// createOpenShiftTagSpecification creates TagSpecification object to add OpenShift tag while
+// launching an instance.
+func (a *AwsProvider) createOpenShiftTagSpecification() ([]*ec2.TagSpecification, error) {
+	infraID, err := a.GetInfraID()
+	if err != nil {
+		return nil, fmt.Errorf("error getting infrastructure ID for the OpenShift cluster, %v", err)
+	}
+	tagslist := map[string]string{
```

Comment:
Perhaps a name change since it's a map not taglist

Reply:
Please don't change to `tagMap` :-) Just go with `tags`. You don't need to express the datastructure being used in the variable name.